docs/interpreter/livy.md

```diff
@@ -144,7 +144,12 @@ Example: `spark.driver.memory` to `livy.spark.driver.memory`
     <td>zeppelin.livy.ssl.trustStorePassword</td>
     <td></td>
     <td>password for trustStore file. Used when livy ssl is enabled</td>
-  </tr>  
+  </tr>
+  <tr>
+    <td>zeppelin.livy.http.headers</td>
+    <td>key_1: value_1; key_2: value_2</td>
```

Comment:
Are ; and : valid separators that would not be present in any header itself?

Reply:
See http://greenbytes.de/tech/webdav/rfc7230.html#rfc.iref.d.2